google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/Entity.java

```diff
@@ -89,4 +89,9 @@ public static Builder newBuilder(Key key, FullEntity<?> copyFrom) {
   public static Entity fromPb(com.google.datastore.v1.Entity entityPb) {
     return new Builder().fill(entityPb).build();
   }
+
+  public static int calculateSerializedSize(BaseEntity<? extends IncompleteKey> e) {
```

Comment:
Please add a javadoc comment to explain the usage of this method (and to help people find it when they google for this type of thing).